src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/InputFileResolverTests.cs

```diff
@@ -136,17 +134,60 @@ public void InputFileResolver_InitializeShouldIgnoreBinFolder(string folderName)
                     { Path.Combine(_filesystemRoot, "TestProject", "TestProject.csproj"), new MockFileData(projectFile)},
                     { Path.Combine(_filesystemRoot, "TestProject", folderName, "somecsharpfile.cs"), new MockFileData("Bytecode") },
                     { Path.Combine(_filesystemRoot, "TestProject", folderName, "subfolder", "somecsharpfile.cs"), new MockFileData("Bytecode") },
-
                 });
 
             var target = new InputFileResolver(fileSystem);
 
-            var result = target.ResolveInput(Path.Combine(_filesystemRoot, "TestProject"), "");
+            var result = target.ResolveInput(Path.Combine(_filesystemRoot, "TestProject"), "", new List<string>());
 
             result.TestProjectPath.ShouldBe(Path.Combine(_filesystemRoot, "TestProject"));
             result.ProjectContents.Children.Count.ShouldBe(1);
         }
 
+        [Fact]
+        public void InputFileResolver_InitializeShouldExcludeSpecifiedFiles()
+        {
+            // the bin, obj and node_modules folders should be skipped
+            string sourceFile = File.ReadAllText(_currentDirectory + "/Initialisation/TestResources/ExampleSourceFile.cs");
+            string projectFile = @"
+<Project Sdk=""Microsoft.NET.Sdk"">
```

Comment:
I've seen this exact string more than once in this test class. It would improve much if this default string could be shared across the class.